sockeye/rnn_attention.py

```diff
@@ -689,16 +748,46 @@ def attend(att_input: AttentionInput, att_state: AttentionState) -> AttentionSta
         return attend
 
 
+@Attention.register(C.ATT_COV)
+class MlpCovAttention(MlpAttention):
+    """
+    MlpAttention with optional coverage config.
+
+    :param input_previous_word: Feed the previous target embedding into the attention mechanism.
+    :param num_hidden: Number of hidden units.
+    :param layer_normalization: If true, normalizes hidden layer outputs before tanh activation.
+    :param config_coverage: Optional coverage config.
```

Comment:
If you really want to have this string inheritance (MLPAttention -> MLPCovAttention), then the coverage config is not optional here.

Reply:
changed